lnwallet/chainfee/rates.go

```diff
@@ -70,6 +70,11 @@ func (s SatPerKWeight) FeePerKVByte() SatPerKVByte {
 	return SatPerKVByte(s * blockchain.WitnessScaleFactor)
 }
 
+// FeePerVByte converts the current fee rate from sat/kw to sat/vb.
+func (s SatPerKWeight) FeePerVByte() SatPerVByte {
+	return SatPerVByte(s * blockchain.WitnessScaleFactor / 1000)
```

Comment:
I think we have a few other areas we do this custom calculation. Let's replace them with this instead? 

EDIT: did a basic sweep, and would seem not.

Reply:
cool found a few other places and replaced.